### ISSUE TITLE
Preference for sources to use

### ISSUE THREAD

**Issue by CatalinFrancu** [dexonline/dexonline#212]
This has now had multiple user requests.
1. Make source selection multiple-choice in the advanced search tab. UI to be determined (it should not be too tall).
2. Allow an anonymous preference and a user preference for multiple source choices (currently the choice only propagates from page to page, but it vanishes when you go to a non-search page).
3. Most likely, when the user sets their choices explicitly in the advanced search tab, it should take precedence over the preferences.
4. The advanced search tab should always stay open for users who have made their choices.

**Comment by sandriu** [comment]
Cumva tichetul ăsta este acoperit de #474 ?

**Comment by CatalinFrancu** [comment]
Nu, par chestii distincte. Din acest tichet rezultă că ar trebui ca selecția surselor să fie multiplă (și în #474).
